tests/common.go

```diff
@@ -811,19 +823,64 @@ func ValidatePureVolumeStatisticsDynamicUpdate(ctx *scheduler.Context, errChan .
 	})
 }
 
-// ValidatePureVolumeSnapshotAndRestore is the ginkgo spec for validating actually creating a FADA snapshot, restoring and verifying the content
-func ValidatePureVolumeSnapshotAndRestore(ctx *scheduler.Context, errChan ...*chan error) {
+// ValidateCSISnapshotAndRestore is the ginkgo spec for validating actually creating a FADA snapshot, restoring and verifying the content
+func ValidateCSISnapshotAndRestore(ctx *scheduler.Context, errChan ...*chan error) {
 	context("For validation of an snapshot and restoring", func() {
+		var err error
 		timestamp := strconv.Itoa(int(time.Now().Unix()))
-		snapShotClassName := PureSnapShotClass + "-" + timestamp
```

Comment:
Any specific reason for changing this? I prefer hyphens personally to be more consistent

Reply:
Discussed w/ En in Slack, this was to make it easier to split and get the timestamp/snapshot class name.